README.rst

```diff
@@ -20,8 +20,8 @@ to identify disks that may need to be replaced soon. You will:
 
 What You Will Need
 ==================
--   `JDK 6 or JDK 7 <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`__
```

Comment:
Please add a space after `==================`, just to be consistent with other READMEs.